presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -88,6 +88,7 @@
     private boolean respectTableFormat = true;
     private boolean immutablePartitions;
     private boolean insertOverwriteImmutablePartitions;
+    private boolean failFastOnInsertIntoImmutablePartitionsEnabled;
```

Comment:
The only concern I have is that this seems to change the default behavior and I'm not sure if we want to start with setting the default value to `true`.

Reply:
I think you are right. Changing it to `true` by default.